client/src/components/Menu/Menu.scss

```diff
@@ -249,19 +247,18 @@
 }
 
 .cms-menu-mobile-overlay {
+  display: block;
   position: absolute;
-  background-color: rgba(0, 0, 0, 0.05);
+  transition: opacity .3s;
+  transform: translateX(-100%);
+  background-color: rgba(0, 0, 0, 0.25);
```

Comment:
Just wondering the reasoning behind the tweaked background-color?

Reply:
because an alpha value of 0.05 is to small, the overlay is almost invisible, so 0.25 could be a good value